### DISCUSSION TITLE
Cannot save credentials for https://signin.aws.amazon.com/signin

### DISCUSSION THREAD

**Discussion by stdedos** [keepassxreboot/keepassxc#10472] [Q&A] [answered]
Is there anything in particular that would make detection for https://signin.aws.amazon.com/signin not work?

Description
KeePassXC integration for modern web browsers
Version
1.9.0.1

KeePassXC - Version 2.7.7
Revision: 68e2dd8

Qt 5.12.8
Debugging mode is disabled.

Operating system: Ubuntu 20.04.6 LTS
CPU architecture: x86_64
Kernel: linux 5.15.0-100-generic

Enabled extensions:
- Auto-Type
- Browser Integration
- Passkeys
- SSH Agent
- KeeShare
- YubiKey
- Secret Service Integration

Cryptographic libraries:
- Botan 2.12.1

**Comment by varjolintu** [comment]
Track this to: https://github.com/keepassxreboot/keepassxc-browser/issues

**Reply by stdedos** [reply, marked as answer]
https://github.com/keepassxreboot/keepassxc-browser/issues/1358#issuecomment-2016794620
